recipes/pytextrank/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "pytextrank" %}
+{% set version = "3.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/pytextrank-{{ version }}.tar.gz
+  sha256: b1bec96442c49984f51283b1f46131ac72908cd0076af790104c1e63b26f155e
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - git
+  host:
+    - python {{ python_min }}
+    - setuptools >=68.0
+    - setuptools-scm >=6.2
+    - wheel >=0.42
+    - pip
+  run:
+    - python >={{ python_min }}
+    - gitpython >=3.1
+    - python-graphviz >=0.13
+    - icecream >=2.1
+    - networkx >=2.6
+    - pygments >=2.7.4
+    - scipy >=1.7
+    - spacy >=3.0
+
+test:
+  imports:
+    - pytextrank
+  commands:
+    - pip check
+  requires:
+    - python {{ python_min }}
+    - pip
+    - git
```

Comment:
Why is `git` required here?

Reply:
Good question. Short answer: because tests don't pass without it.

GitPython is used to get the version and is imported here: https://github.com/DerwenAI/pytextrank/blob/e119be8516b4414734d89e73cd961b85a2e67ba0/pytextrank/version.py#L14

Perhaps `from git import Repo` should be moved into the try-except (or use e.g. `setuptools-scm` or `setuptools-git-versioning` or `importlib.metadata.version`).

If/when this gets cleaned up upstream, we will remove `git` from requires sections.